### PR TITLE
update device ID map <-> name

### PR DESCRIPTION
**Pull request by mlp6**
Several devices had names and IDs that did not match what the particle cloud had registered; these have been updated.

@aforbis-stokes please confirm that remapping the names to correctly reflect the device IDs won't break anything on your end.

@suyashkumar same comment as above, but w/r to the database; do you store by name or ID?

### CODE REVIEW

**Comment by suyashkumar** [comment]
@mlp6: Yes these location keys are stored with relevant data in the database. As I mentioned to Aaron, I think the names here should be the human readable names because we can map devices back to the particle cloud using the coreid. These names should define the location **not** the specific particle device since they are used in the UI. 

Any change to these names will also require a db migration to migrate old records to the new name.

**Comment by mlp6** [comment]
That is what I thought... the issue that reared its head is that my old mapping by name pointed to a different device ID, that had a different system firmware version running, which caused some failures.

@aforbis-stokes The issue stems from the number in the location name... what was showing up on the particle dashboard differed from the name on our webserver.  We might want to reject this pull request if we don't want to change names, but we should then rename the devices on the particle dashboard and remove the old IDs.

Let me know how you would like to proceed.

**Comment by suyashkumar** [comment]
tl;dr we need to keep database records from Kenya-north location grouped together even if the underlying particle device changes. This is currently done based on name. 

So if we rename, I think we should rename these into "Kenya-North" and "Kenya-Central" type names and I can do a db migration based on that. Let me know your thoughts folks!

**Comment by mlp6** [comment]
you can see why I didn't blindly merge this change in.  :)

**Comment by mlp6** [comment]
@aforbis-stokes to make life easier in the short-term, I can remap the IDs to what the dashboard has been displaying; does that work for you?

**Comment by aforbis-stokes** [comment]
The naming method currently used is so that I can match the ID on the physical Particle to the correct coreID; however, that method no longer works when changing out damaged Particles while trying to maintain the database records. @suyashkumar and I decided to keep the existing names until we found a good time to change all the names to a constant name. I am ok if that happens now. 
Let's drop ADPL from all names and the name. Devices should be "Kenya-Central" "Kenya-North" "Duke" "India" "Philippines-Subayon" "Philippines-Sagay".

**Review by aforbis-stokes — COMMENTED** [review]
ADPLDuke6191 not in use. Comment out.
ADPLKenyaC6304 not in use.
ADPLIndia3847 not in use.
Both Philippines can be changed now, will lose no data:
ADPL..Sagay.. -> Philippines-Sagay
ADPL..Subayon.. -> Philippines-Subayon
